clo-workflows/Scripts/src/store/ClientStore.ts

```diff
@@ -1,146 +1,96 @@
-import { RootStore } from "./RootStore"
 import { action, ObservableMap, observable, runInAction, computed } from "mobx"
-import { FormEntryType, CloRequestElement } from "../model/CloRequestElement"
 import { autobind } from "core-decorators"
+
+import { FormEntryType, CloRequestElement, PROJECT_TYPES, WORK_TYPES } from "../model/CloRequestElement"
 import { IFormControl } from "../model/FormControl"
-import { validateFormControl } from "../utils"
-import { WORK_TYPES, PROJECT_TYPES } from "../model/CloRequestElement"
-import { IView } from "../model/View"
-import { IDataService } from "../service/dataService/IDataService"
 import { getView } from "../model/loader/resourceLoaders"
+import { IDataService } from "../service/dataService/IDataService"
+import { validateFormControl } from "../utils"
+import { RootStore } from "./RootStore"
+import { User } from "../model/User"
+import { Selection, SelectionMode, MessageBarType } from "office-ui-fabric-react"
+import { IMessageProps } from "../component/Message"
```

Comment:
So actually we can't import IMessageProps and use it here (sorry, you probably got this from an old branch of mine) because it breaks tests. So just cast message stuff to any and don't import anything from Message.tsx. Not very typesafe but its the fix for now